README.md

```diff
@@ -166,23 +166,16 @@ ssh:
       - Accept-Language
       - Content-Language
       - Origin
-    #  - Content-Type
-    #  - X-Requested-With
-    #  - User-Agent
-    #  - Authorization
-    #  - Access-Control-Request-Method
 
     # The allowed cross-origin URLs
-    # allowed_origins:
-    #    - *
+    allowed_origins:
+      - "" # do not allow any origins
```

Comment:
We don't need an empty string, an empty list should have the same meaning, no?

Reply:
in gorilla by default an empty list equals to "*"
https://github.com/gorilla/handlers/blob/9c61bd81e701cf500437e1b516b675cdd3b73ca7/cors.go#L117

Reply:
We should always prepend the public http origin to the list so that it has at least one item

Reply:
Before this PR even the public url was not an allowed origin. So I set "" and do not allow any origins to match the original behaviour 

If you're okay to make the default less safe but more convenient, then yes, I can prepend `c.HTTP.PublicURL` to `c.HTTP.CORS.AllowedOrigins` in the `Config.Validate()` function.

Reply:
Right, the Soft Serve public URL would usually be the origin where we expect requests to come from. IMO, it should always be included in the list of allowed origins or be defined in an allowed origins validator.